packages/screens/DAppStore/query/getFromAirTable.tsx

```diff
@@ -0,0 +1,74 @@
+import Airtable from "airtable";
+
+import { dAppGroup, dAppType } from "../types";
+
+export async function getFromAirTable(): Promise<dAppGroup> {
```

Comment:
I don't think we should use airtable for this as a dapp has to have code with it so for now we need to update this repo whenever we want to implement a dapp

Reply:
left some notes at discord